src/components/Calculator.js

```diff
@@ -12,6 +12,7 @@ const Calculator = () => {
   return (
     <>
       <div className="container">
+        <h2 className="calc-title">Lets do some Math!</h2>
         <div className="display" id="display">
           {calc.total}
           {' '}
```

Comment:
[REQUIRED] Kindly adjust the layout to match the wireframe.

![image](https://user-images.githubusercontent.com/62938735/214819570-77925d9c-5756-423c-bff4-08e9ab817ebc.png)